src/usersdialog.cpp

```
@@ -202,11 +211,15 @@ void UsersDialog::itemChange(QListWidgetItem *item) {
  * @param filter
  */
 void UsersDialog::populateList(const QString &filter) {
-  if (filter != m_lastFilter) {
-    m_lastFilter = filter;
-    m_cachedNameFilter = QRegularExpression(
-        QRegularExpression::wildcardToRegularExpression("*" + filter + "*"),
+  QString patternString = "*" + filter + "*";
+  if (m_cachedPatternString != patternString) {
+    QRegularExpression re(
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)